autoscheduler/frontend/src/index.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import * as ReactDom from 'react-dom';
 
-import App from './App';
+import App from './App/App';
```

Comment:
The only reason I hesitated to do that is because App feels more like a root thing than it does like a component to me. That said, I'm okay with either, so I'll let @ryan-conn break the tie

Reply:
I think putting it in components makes it look more organized, and even though it's the root of our website, it's still technically a component, so I agree with Gannon